{{cookiecutter.project_dirname}}/scripts/check.sh

```diff
@@ -6,5 +6,6 @@ python3 -m black --check .
 python3 -m isort --check .
 python3 -m flake8
 python3 -m mypy .
+python3 manage.py check --deploy --fail-level=WARNING --configuration=Remote
```

Comment:
I would raise the `--fail-level` to `ERROR`

Reply:
This is debatable: remarkable warnings are raised that should be at least taken into account.

Reply:
Try running the command in an actual project.